### ISSUE TITLE
Filter Streaming Service(s)/Network(s)

### ISSUE THREAD

**Issue by thegregstengel** [sct/overseerr#2719]
### Description

I'd like to have the ability to filter out specific streaming services. I have a subscription to Netflix, so ideally I could filter out movies/shows that are on Netflix already.

### Desired Behavior

Not sure the best implementation, but maybe under settings, there is a streaming service or networks setting and you could include or exclude them from "Discover"?

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by pironic** [comment]
Seems like a duplicate of: https://github.com/sct/overseerr/issues/835

**Comment by ZedNetzAU** [comment]
I don't think this is a duplicate because #835 is about limiting a users ability to request content from a service. Where as this issue is more about being about to filter out certain services when looking for content. For me that would be perfect seeing as sometimes I have subs and sometimes I dont. If I am looking for new content it would be great to just say "Ok., I have Netflix at the moment so I don't want to see that".

**Comment by 2fst4u** [comment]
It does sound like just a functionally different method of achieving the same outcome as it though. 

Just hiding things entirely could have a couple of issues: 
- A user might get confused why they can't find something and annoy you as the admin further about why they can't see it, and ask you directly (defeating the purpose of the app)
- The APIs for streaming service content can sometimes have false positives and negatives and it could hide things that haven't had time to propagate through the API that you might want downloaded on the very day it happened to drop off a streaming service. This would mean you have to go and turn off the hiding of content just to download these.

The other issue would present it like "here's the item you're looking for, but the request button is greyed out because it's on streaming service X", making it a less confusing UX.

Just my two cents.

**Comment by 2fst4u** [comment]
Bump

**Comment by dinosoup1** [comment]
I would like this feature to still be considered. I would love to be able to filter out content that is available on the streaming services I actually do pay for. I don't want to download something that I can just stream from one of the services. This is not some kind of global catch all that hides things from everyone, just a "show my movies that are not accessible via the selected streaming services" (i.e. checkbox for netflix, check box for hbo, etc).

**Comment by 2fst4u** [comment]
bump

**Comment by 2fst4u** [comment]
Bump

**Comment by 2fst4u** [comment]
Bump

**Comment by thegregstengel** [comment]
Bump
